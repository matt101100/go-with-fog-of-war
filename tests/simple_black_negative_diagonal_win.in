who
place A5
place A1
place B4
place A2
place C3
place A3
place D2
place A4
place E1
