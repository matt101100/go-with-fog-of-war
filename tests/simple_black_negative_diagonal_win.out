B
Black wins!
A5A1B4A2C3A3D2A4E1
Thank you for playing!
